Source/CBL_Router.m

```diff
@@ -204,6 +204,17 @@ - (CBLQueryOptions*) getQueryOptions {
     options->group = [self boolQuery: @"group"];
     options->content = [self contentOptions];
 
+    // Stale options (ok or update_after):
+    NSString *stale = [self query: @"stale"];
+    if (stale) {
+        if ([stale isEqualToString:@"ok"])
+            options->indexUpdateMode = kCBLUpdateIndexNever;
+        else if ([stale isEqualToString:@"update_after"])
+            options->indexUpdateMode = kCBLUpdateIndexAfter;
+        else
+            return nil;
```

Comment:
Oh, I didn't know it rejected other values. I know that Couchbase Server accepts `stale=false` as well, but that's probably just because it defaults (defaulted?) to `update_after`. Never mind!